lib/ReactViews/HOCs/measureElement.tsx

```diff
@@ -1,13 +1,29 @@
 import React from "react";
 import debounce from "lodash-es/debounce";
 
-type Measurements =
-  | "widthFromMeasureElementHOC"
-  | "heightFromMeasureElementHOC";
+const getDisplayName = <P extends React.ComponentProps<any>>(
+  WrappedComponent: React.ComponentClass<P>
+) => {
+  return WrappedComponent.displayName || WrappedComponent.name || "Component";
+};
```

Comment:
This file was TSified twice. I am using my version here as its more prescriptive and more likely to find problems with components wrapped in `measureElement` using the type system.